scss/utilities/mixins/molecules/_footer-menus.scss

```diff
@@ -0,0 +1,37 @@
+@charset "UTF-8";
```

Comment:
I think this file has too many mixins and could be reduced to 1 or 2. Even better, would be to create a generic mixin for a vertical list that could be re-used instead of something specific for the footer.